src/scenes/Languages/Overview/LanguageOverview.graphql

```diff
@@ -0,0 +1,37 @@
+query Language($input: ID!) {
+  language(id: $input) {
+    id
```

Comment:
I think a `languageDetail` fragment would be good, that way the mutation can return the same thing. I think that's important for apollo to automatically update cache state.

Reply:
This is changed to use a fragment